cmd/app/start/start.go

```diff
@@ -89,183 +89,183 @@ func (s *Start) CreateStartCommand() *cobra.Command {
 		Int64P(
 			"monitor-retry-count", "m",
 			s.configs.MonitorRetryInSeconds,
-			"The number of retries for the monitor.",
+			"The number of retries for the monitor",
 		)
 
 	startCmd.PersistentFlags().
 		StringP(
 			"output-format", "o",
 			s.configs.PrintOutputType,
-			"The format for the output to be shown. Options are: text (stdout), json, sonarqube",
+			`Output format of analysis ("text"|"json"|"sonarqube"). For json and sonarqube --json-output-file is required`,
 		)
 
 	startCmd.PersistentFlags().
 		StringSliceP(
 			"ignore-severity", "s",
 			s.configs.SeveritiesToIgnore,
-			"The level of vulnerabilities to ignore in the output. Example: -s=\"LOW, MEDIUM, HIGH\"",
+			`The level of vulnerabilities to ignore in the output ("LOW"|"MEDIUM"|"HIGH"). Example: -s="LOW, HIGH"`,
 		)
 
 	startCmd.PersistentFlags().
 		StringP(
 			"json-output-file", "O",
 			s.configs.JSONOutputFilePath,
-			"If your pass output-format you can configure the output JSON location. Example: -O=\"/tmp/output.json\"",
+			`Output file to write analysis result. This flag should be used with --output-format`,
 		)
 
 	startCmd.PersistentFlags().
 		StringSliceP(
 			"ignore", "i",
 			s.configs.FilesOrPathsToIgnore,
-			"Paths to ignore in the analysis. Example: -i=\"/home/user/project/assets, /home/user/project/deployments\"",
+			`Paths to ignore in the analysis. Example: -i="/path/to/ignore, **/*_test.go, **/assets/**"`,
 		)
 
 	startCmd.PersistentFlags().
 		StringP(
 			"horusec-url", "u",
 			s.configs.HorusecAPIUri,
-			"The Horusec API address to access the analysis engine",
+			"The Horusec server address to send analysis results",
 		)
 
 	startCmd.PersistentFlags().
 		Int64P(
 			"request-timeout", "r",
 			s.configs.TimeoutInSecondsRequest,
-			"The timeout threshold for the request to the Horusec API",
+			"The timeout threshold for the request to the Horusec server. The minimum time is 10",
```

Comment:
Should we inform also the default value?

Reply:
The default value is already filled by cobra.
```bash
Start the Horusec analysis in the current path

Usage:
  horusec start [flags]

Examples:
horusec start

Flags:
  -t, --analysis-timeout int                 The timeout threshold for the Horusec CLI wait for the analysis to complete. The minimum time is 10 (default 600)
  -a, --authorization string                 Authorization token to use on Horusec server. Read more: https://docs.horusec.io/docs/tutorials/how-to-create-an-authorization-token (default "00000000-0000-0000-0000-000000000000")
  -C, --certificate-path string              Path to certificate of authority. Example -C="example/ca.crt"
  -P, --container-bind-project-path string   Project path in host to be used on Docker when running Horusec inside a container
  -c, --custom-rules-path string             Path with custom rules that should be used by Horusec engine
  -D, --disable-docker                       Run Horusec without docker. If enabled it will only run the following tools: horusec-csharp, horusec-kotlin, horusec-java, horusec-kubernetes, horusec-leaks, horusec-javascript, horusec-dart, horusec-nginx
  -G, --enable-commit-author                 Enable to search commit author of vulnerabilities
      --enable-git-history                   Run Gitleaks and search for vulnerabilities in all git history of the project https://github.com/zricethezav/gitleaks
  -w, --enable-owasp-dependency-check        Run Owasp Dependency Check tool https://github.com/jeremylong/DependencyCheck
  -j, --enable-shellcheck                    Run ShellCheck tool https://github.com/koalaman/shellcheck
  -F, --false-positive strings               Ignore a vulnerability by hash and set it to be false positive. Example -F="hash1, hash2"
      --headers stringToString               Custom headers to send on request to Horusec API. Example --headers='{"X-Auth-Service": "value"}' (default [])
  -h, --help                                 help for start
  -u, --horusec-url string                   The Horusec server address to send analysis results (default "http://0.0.0.0:8000")
  -i, --ignore strings                       Paths to ignore in the analysis. Example: -i="/path/to/ignore, **/*_test.go, **/assets/**" (default [*tmp*,**/.vscode/**])
  -s, --ignore-severity strings              The level of vulnerabilities to ignore in the output ("LOW"|"MEDIUM"|"HIGH"). Example: -s="LOW, HIGH" (default [INFO])
  -I, --information-severity                 Enable information severity vulnerabilities. Information vulnerabilities can contain a lot of false positives
  -S, --insecure-skip-verify                 Disable the certification validation. PLEASE, try not to use it
  -O, --json-output-file string              Output file to write analysis result. This flag should be used with --output-format
  -o, --output-format string                 Output format of analysis ("text"|"json"|"sonarqube"). For json and sonarqube --json-output-file is required
  -p, --project-path string                  Path to run an analysis. If this value is not passed, Horusec will ask if you want to run the analysis in the current directory (default "/home/user/horusec")
  -n, --repository-name string               Send repository name to Horusec server (default "horusec")
  -r, --request-timeout int                  The timeout threshold for the request to the Horusec server. The minimum time is 10 (default 300)
  -e, --return-error                         Return exit code 1 if found vulnerabilities. Default value is false (exit code 0)
  -R, --risk-accept strings                  Ignore a vulnerability by hash and set it to be risk accept. Example -R="hash1, hash2"
      --show-vulnerabilities-types strings   Show vulnerabilities by types ("Vulnerability"|"Risk Accepted"|"False Positive"|"Corrected"). Example --show-vulnerabilities-types="Vulnerability, Risk Accepted" (default [Vulnerability])

Global Flags:
      --config-file-path string   Path of the configuration file (default "/home/user/horusec/horusec-config.json")
  -l, --log-file-path string      Path of log file (default "/tmp/horusec-2021-12-06-09-54-45.log")
      --log-level string          Set log level ("panic"|"fatal"|"error"|"warn"|"info"|"debug"|"trace") (default "info")

```